polling/pcc_policy_config.go

```diff
@@ -120,11 +120,15 @@ func hasDefaultQosFlow(flows []models.FlowInformation) bool {
 
 func makeQosDesc(id int64, pccQos nfConfigApi.PccQos) models.QosData {
 	qos := models.QosData{
-		QosId:   strconv.FormatInt(id, 10),
-		Var5qi:  pccQos.FiveQi,
-		MaxbrUl: pccQos.MaxBrUl,
-		MaxbrDl: pccQos.MaxBrDl,
-		Arp:     &models.Arp{PriorityLevel: pccQos.Arp.PriorityLevel},
+		QosId:  strconv.FormatInt(id, 10),
+		Var5qi: pccQos.FiveQi,
+		Arp:    &models.Arp{PriorityLevel: pccQos.Arp.PriorityLevel},
+	}
+	if MaxbrUl, ok := pccQos.GetMaxBrUlOk(); ok {
+		qos.MaxbrUl = *MaxbrUl
+	}
+	if MaxbrDl, ok := pccQos.GetMaxBrDlOk(); ok {
+		qos.MaxbrDl = *MaxbrDl
```

Comment:
Why not using the Getter/Setter methods already part of the codebase?
```suggestion
		Var5qi: pccQos.GetFiveQi(),
		Arp:    &models.Arp{
			PriorityLevel: pccQos.GetArp().PriorityLevel,
		},
	}
	if pccQos.HasMaxBrUl() {
		qos.MaxbrUl = pccQos.GetMaxBrUl()
	}
	if pccQos.HasMaxBrDl() {
		qos.MaxbrDl = pccQos.GetMaxBrDl()
```